src/Components/Hero/index.jsx

```diff
@@ -1,15 +1,18 @@
-function Hero() {
+function Hero({ heroText, currentId }) {
   return (
     <section className="w-full mt-4 md:px-4 h-60">
-        <div className="bg-[url(/burgers.jpg)] w-full h-full bg-cover bg-center content-center">
-          <div className="mx-auto items-center bg-white bg-opacity-70 p-6 max-md:w-11/12 w-3/5 max-w-3xl">
-            <h1 className="text-cyan-500 text-5xl text-center">Food. Delivered.</h1>
+      <div className="bg-[url(/burgers.jpg)] w-full h-full bg-cover bg-center content-center">
+        <div className="mx-auto items-center bg-white bg-opacity-70 p-6 max-md:w-11/12 w-3/5 max-w-3xl">
+          <h1 className="text-cyan-500 text-5xl text-center">{heroText}</h1>
+          {!currentId && 
```

Comment:
Why don't you just check if the `heroText` is something or empty string? You don't *actually use* the `currentId` for anything in this component, so why also pass it in as a prop?

Reply:
Done